source/documentation/deploying-an-app/multicontainer-app-deploy.md

```diff
@@ -87,6 +87,28 @@ You should be able to view the application in your browser at:
 
 It should behave in the same way as when you were running it locally via docker-compose.
 
+### k8s_additional_resources
```

Comment:
I'm not sure about this title "k8s_additional_resources" That isn't really what this document is about - monitoring, cron jobs and custom alerts aren't specific to k8s.

How about something like "Further Development" or "Additional Exercises" ?

Reply:
Agree with you and renamed it as "Further Development"